src/stories/Blocks/material-banner/MaterialBanner.stories.tsx

```diff
@@ -7,7 +7,10 @@ export default {
   component: MaterialBanner,
   decorators: [withDesign],
   argTypes: {},
-  parameters: {},
+  parameters: {
+    // Notifies Chromatic to pause the animations when they finish for the specific story.
+    chromatic: { pauseAnimationAtEnd: true },
```

Comment:
This says what the configuration does but not *why* we think it is important to add/have it. The commit comment for 45c4235b7881de8c17f836b90e3caf9e81444292 mentions some of it. Perhaps that is sufficient compared to duplicating this comment across all stories. Perhaps it is something that we can define globally once?